commands/util/import.go

```diff
@@ -189,6 +190,14 @@ func ImportChain(ctx context.Context, r repo.Repo, fname string, snapshot bool)
 		return fmt.Errorf("importing chain failed: %w", err)
 	}
 
+	// The cst.Import function will only backfill 1800 epochs of tipsetkey,
+	// as determined by the value specified in TipsetkeyBackfillRange.
```

Comment:
this line might be confusing since we have `backfillTipsetKeyRange`. No need to mention the variable in `cst.import`.

Reply:
removed